### DISCUSSION TITLE
MONIKA IS BEAUTIFUL

### DISCUSSION THREAD

**Discussion by CharlyIsHungry** [Monika-After-Story/MonikaModDev#8790] [appreciation]
OH MY GOD THE COSTUME IS SO PRETTY AND I LOVE THE DRAWING AND MONIKA HAVE MORE POSES AND SHE'S BEAUTIFUL I LOVE HALLOWEEN EVEN MORE NOW I LOVE IT I HOPE WE CAN KEEP THIS AFTER HALLOWEEN BECAUSE SHE'S SO BEAUTIFUL IN IT

**Comment by TheNxtCypher** [comment]
I agree with everything said in this issue

**Comment by Mcmanybucks** [comment]
We absolutely need more CG of this caliber, it's amazing!

**Comment by AsuraShun** [comment]
it was the best thing I ever watched in my entire life!
